File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        
        // HashMap<Character, Integer> alpha_dict = new HashMap<Character, Integer>();
	    // for(int i=0;i<26;i++) {
	    //     alpha_dict.put((char)(i+97), 0);
	    // }
	    // for(int i=0; i<sentence.length(); i++) {
	    //     alpha_dict.put(sentence.charAt(i), alpha_dict.get(sentence.charAt(i))+1);
	    // }
	    // return !alpha_dict.containsValue(0);


        char[] alpha = new char[26]; 
	    for(int i=0; i<sentence.length(); i++)
	        alpha[((int)sentence.charAt(i))-97] = sentence.charAt(i);

	    for (int i=0; i<alpha.length; i++)
	        if(alpha[i] == 0)
	            return false;
	    return true;        

    }
}

File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:

        # Apporach 1 Using Hashing
        # alpha_dict = {}
        # for ele in range(0,26):
        #     alpha_dict[chr(ele+97)] = 0
            
        # for ele in sentence:
        #     if ele in alpha_dict:
        #         alpha_dict[ele] += 1

        # return False if 0 in alpha_dict.values() else True

        # Approach 2 Using Lists
        alpha_list = [0 for i in range(26)]

        for ele in sentence:
            alpha_list[ord(ele)-97] = ele
        
        return False if 0 in alpha_list else True